smoketests/__init__.py

```diff
@@ -0,0 +1,195 @@
+from pathlib import Path
+import unittest
+import tempfile
+import os
+import re
+import string
+import random
+import string
+import subprocess
+import json
+import sys
+import shutil
+
+TEST_DIR = Path(__file__).parent
```

Comment:
All path manipulations should use `pathlib` so is possible to run this on Windows.

Reply:
I'm pretty sure I already did that; did you notice somewhere where that's not the case?

Reply:
Like in `SPACETIME_BIN = STDB_DIR / "target/debug/spacetime"`?

(Or this is something new Python covers already)?